### PR TITLE
Changed Azure interpret nb to run using this repo with dashboard

### PR DESCRIPTION
**Pull request by bethz**
- changed imports to use updated reference
- added dashboard
- using prerelease dashboard (not pypi)

### CODE REVIEW

File: notebooks/explain-binary-classification-local.ipynb [hunk 1]
```diff
@@ -0,0 +1,1031 @@
...
+    "Copyright (c) Microsoft Corporation. All rights reserved.\n",
+    "\n",
+    "Licensed under the MIT License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![Impressions](https://PixelServer20190423114238.azurewebsites.net/api/impressions/MachineLearningNotebooks/how-to-use-azureml/explain-model/tabular-data/explain-binary-classification-local.png)"
```
Comment (imatiach-msft) [inline notebooks/explain-binary-classification-local.ipynb]: remove?

File: notebooks/explain-binary-classification-local.ipynb [hunk 2]
```diff
@@ -0,0 +1,1031 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Could not import lightgbm, required if using LGBMExplainableModel\n",
```
Comment (imatiach-msft) [inline notebooks/explain-binary-classification-local.ipynb]: clear the output cells

File: notebooks/explain-binary-classification-local.ipynb (hunk 1 above)
Comment (bethz) [reply notebooks/explain-binary-classification-local.ipynb]: done.
v. good one to note as it was "invisible" when I was using the file.

File: notebooks/explain-binary-classification-local.ipynb (hunk 2 above)
Comment (bethz) [reply notebooks/explain-binary-classification-local.ipynb]: done.

**Pull request merged**